frontend/src/components/teams/teamDetailPanel.css

```
@@ -152,3 +152,89 @@
+  font-weight: 600;
+}
+
+.team-roster-list {
+  max-height: 320px;
+  overflow-y: auto;
+  scrollbar-width: thin;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+
+  padding-right: 6px;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)